stable/database/values.yaml

```diff
@@ -361,6 +362,26 @@ database:
   isManualVolumeProvisioning: false
   isRestore: false
 
+
 openshift:
   # change this to true if you want to deploy using OpenShift proprietary features
   enabled: false
+
+insights:
+  # disable insights by setting insights.enabled=false or admin.insights="" or admin.insights=null
+  # enabled here as a super values file that works for both admin and database helm charts can set
+  # to false to disable for both helm charts.
+  enabled: false
+  image:
+    registry: docker.io
+    repository: nuodb/nuodb-collector
+    tag: 1.0.1
```

Comment:
this and other places. Should we hardcode the version of the collector same as we do the DB, or should we use `latest`? I have no strong preference, but I wanna call it out explicitly.